internal/pkg/term/progress/ecs.go

```diff
@@ -21,6 +23,10 @@ const (
 	maxServiceEventsToDisplay = 5 // Total number of events we want to display at most for ECS service events.
 )
 
+const (
+	maxStoppedTasksToDisplay = 2
```

Comment:
Why choose specifically 2 tasks to display?

Reply:
The main reason for choosing only 2 tasks is in case of long task stopped reason for example(unable to pull secrets reason), the terminal will be filled with only the task stopped reasons and the user will not be able to see the other stack events in the progress tracker if the number is greater than 2